app/utils/number.ts

```diff
@@ -0,0 +1,42 @@
+import BigNumber from 'bignumber.js'                                                        
+
+export const localeNumberString = (value: BigNumber | string | number): string => {
+  if (!value) return '0'
+  const origin = typeof value === 'string' || typeof value === 'number' ? new BigNumber(value) : value
+  const bigValue = origin.abs()
+  if (bigValue.isNaN()) {
+    return '0'
+  }
+  if (bigValue.isLessThan(1) && bigValue.abs().isGreaterThan(0)) {
```

Comment:
`bigValue` is non-negative

Reply:
Why returned without `+/-` sign while the sign is returned along with the value in the following logic